### ISSUE TITLE
No indices for p_dc and q_dc in ac model

### ISSUE THREAD

**Issue by Wikunia** [lanl-ansi/PowerModels.jl#190]
If I run 

```
pm = build_generic_model("data/pglib_opf_case5_pjm.m", ACPPowerModel, PowerModels.post_ots)
m = pm.model
```
and try to print the model using `print(m)` I get:

```
Got an exception of type ArgumentError outside of a @test
  ArgumentError: collection must be non-empty
  Stacktrace:
   [1] first(::JuMP.ValueIterator{JuMP.Variable,1}) at ./abstractarray.jl:153
   [2] model_str(::Type{T} where T, ::JuMP.Model, ::Dict{Symbol,String}) at /home/ole/.julia/v0.6/JuMP/src/print.jl:214
```

The problem seems to be that the indices of `p_dc` and `q_dc` are empty.

```
Any[0 ≤ line_z[l] ≤ 1, integer, ∀ l ∈ {4,2,…,6,1}, va[i] ∀ i ∈ {4,2,3,5,1}, 0.9 ≤ vm[i] ≤ 1.1 ∀ i ∈ {4,2,3,5,1}, 0 ≤ pg[i] ≤ … ∀ i ∈ {4,2,3,5,1}, … ≤ qg[i] ≤ … ∀ i ∈ {4,2,3,5,1}, … ≤ p[(l, i, j)] ≤ … ∀ (l, i, j) ∈ {(4, 2, 3),(2, 1, 4),…,(6, 5, 4),(1, 2, 1)}, … ≤ q[(l, i, j)] ≤ … ∀ (l, i, j) ∈ {(4, 2, 3),(2, 1, 4),…,(6, 5, 4),(1, 2, 1)}, p_dc (no indices), q_dc (no indices)]#Variables: 50
```

Maybe it is possible to test if there are indices before adding the variables to the model.

Printing the model isn't important but sometimes nice to have ;)

**Comment by ccoffrin** [comment]
Good catch.  I have noticed it my self.  This happens in networks that do not feature dc lines.

My interpretation is that this is a bug in JuMP's print function, but I am open to discussion @mlubin, et. al.

**Comment by mlubin** [comment]
Possibly, but printing is going to change completely with JuMP 0.19, so best to revisit after that.

**Comment by ccoffrin** [comment]
Sounds good to me.

**Comment by ccoffrin** [comment]
This seems to be resolved in v0.18.2, will assume no regression will occur in JuMP v0.19.
